tests/requirements.txt

```diff
@@ -1,12 +1,9 @@
-https://github.com/divio/django-cms/archive/release/4.0.x.zip
 coverage
 djangocms_helper
 flake8
 isort
-djangocms_file
-djangocms_picture
-djangocms_audio
-djangocms_video
-https://github.com/divio/djangocms-versioning/archive/master.zip
```

Comment:
ci is failing because versioning has been removed as the tox uses this.

Reply:
I'll see if this makes a difference. Versioning should be installed using setup.py